<launch> 
    <arg name="gui" default="true"/>
    <arg name="args" default=""/>
    <!-- This version uses the openai_ros environments -->
    <rosparam command="load" file="$(find my_parrotdrone_openai_example)/config/parrotdrone_openai_qlearn_params_v2.yaml" />
    <!-- Launch the training system -->
    <node pkg="my_parrotdrone_openai_example" name="parrotdrone_goto_qlearn" type="start_qlearning_v2.py" output="screen"/>

    <!-- start gui -->
    <group if="$(arg gui)">
    <node name="gazebo_gui" pkg="gazebo_ros" type="gzclient" respawn="false" output="screen"/>
  </group>
</launch>

  